ssl/record/rec_layer_s3.c

```diff
@@ -726,21 +727,39 @@ int do_ssl3_write(SSL *s, int type, const unsigned char *buf,
         align = (size_t)SSL3_BUFFER_get_buf(wb) + 2 * SSL3_RT_HEADER_LENGTH;
         align = SSL3_ALIGN_PAYLOAD - 1 - ((align - 1) % SSL3_ALIGN_PAYLOAD);
 #endif
-        outbuf[0] = SSL3_BUFFER_get_buf(wb) + align;
         SSL3_BUFFER_set_offset(wb, align);
+        if (!WPACKET_init_static_len(&pkt[0], SSL3_BUFFER_get_buf(wb),
```

Comment:
Yes they are the same, but in my mind it makes it a lot clearer that you are referring to a single entity and not the array as a whole. Especially when you mix it in scenarios where on other lines we refer to a single entity which is not at index 0. I dislike the inconsistency.